core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -63,19 +71,58 @@ object Constant {
  * http://docs.scala-lang.org/style/naming-conventions.html
  * i.e. upper camel case.
  */
-object Property {
+object Property extends Enumeration {
+
+  final class ValueAlt(name: String, altName: String)
+      extends Property.Val(name) {
+
+    def getOption(conf: SparkConf): Option[String] = if (altName == null) {
+      conf.getOption(name)
+    } else {
+      conf.getOption(name).orElse(conf.getOption(altName))
```

Comment:
Should we throw error here in case a property is present with both name and altName? If name and altName are both given by the user then altName will never be picked up. Might lead to some unintentional configuration?